### ISSUE TITLE
Problem in loss function 

### ISSUE THREAD

**Issue by mephisto28** [cloneofsimo/minRF#5]
https://github.com/cloneofsimo/minRF/blob/72feb0c87d435e9f9d220f34f348ed66c0b6ccec/advanced/main.py#L60

I guess the loss should be `batchwise_mse = ((z1 - x - vtheta * texp) ** 2).mean(dim=list(range(1, len(x.shape))))`... is it?

The current code says the predict target in training should be the distance other than velocity. If not changing the training target the inference code could be modified by dividing the predicted value by timestep to gain the velocity.

**Comment by cloneofsimo** [comment]
Sorry what?

**Comment by mephisto28** [comment]
> Sorry what?

If `mse = (model_pred - x) ** 2`, it should be `model_pred = z1 - vtheta * t` instead of `model_pred = z1 - vtheta` ... is it?

**Comment by cloneofsimo** [comment]
nah
